core/domain/user_jobs_one_off_test.py

```diff
@@ -31,6 +31,109 @@
 from core.tests import test_utils
 
 
+class UserContributionsOneOffJobTests(test_utils.GenericTestBase):
+    """Tests for the one-off dashboard subscriptions job."""
+    EXP_ID_1 = 'exp_id_1'
+    EXP_ID_2 = 'exp_id_2'
+    USER_A_EMAIL = 'a@example.com'
+    USER_A_USERNAME = 'a'
+    USER_B_EMAIL = 'b@example.com'
+    USER_B_USERNAME = 'b'
+    USER_C_EMAIL = 'c@example.com'
+    USER_C_USERNAME = 'c'
+    USER_D_EMAIL = 'd@example.com'
+    USER_D_USERNAME = 'd'
+
+    def _run_one_off_job(self):
+        """Runs the one-off MapReduce job."""
+        job_id = user_jobs_one_off.UserContributionsOneOffJob.create_new()
+        user_jobs_one_off.UserContributionsOneOffJob.enqueue(job_id)
+        self.assertEqual(
+            self.count_jobs_in_taskqueue(
+                queue_name=taskqueue_services.QUEUE_NAME_DEFAULT),
+            1)
+        self.process_and_flush_pending_tasks()
+
+    def setUp(self):
+        super(UserContributionsOneOffJobTests, self).setUp()
+
+        self.signup(self.USER_A_EMAIL, self.USER_A_USERNAME)
+        #User A has no created or edited explorations 
```

Comment:
All this commentary might as well go above the previous line, so that you have one block of commentary and then all the commands that relate to it. It seems weird to split the latter into 1 line before and 7 lines after.